endstate_correction/tests/simulation/charmm.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from importlib_resources import files
+
+import endstate_correction
+from endstate_correction.protocol import BSSProtocol
+from endstate_correction.simulation import EndstateCorrectionCHARMM, EndstateCorrectionAMBER
+from endstate_correction.simulation.base import EndstateCorrectionBase
+from endstate_correction.topology import CHARMMTopology, AMBERTopology
+
+
+class TestEndstateCorrectionCharmm():
+    @staticmethod
+    @pytest.fixture(scope="module")
+    def bss_protocol():
+        protocol = BSSProtocol(
+            timestep=1,
```

Comment:
Could you please add these tests to the `test_charmm.py`? 
I totally missed these new tests, and they weren't picked up by `pytests` since the file name doesn't start with `test_`.